client/serve/serve.json

```diff
@@ -112,6 +112,64 @@
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-is-the-aria-hidden-attribute",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-is-the-aria-hidden-attribute"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-is-the-aria-expanded-attribute",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-is-the-aria-expanded-attribute"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-is-the-aria-live-attribute",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-is-the-aria-live-attribute"
+    },
+    {
+      "source": "learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-are-some-common-aria-states-used-on-custom-control-elements",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-are-some-common-aria-states-used-on-custom-control-elements"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-is-the-aria-controls-attribute",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-is-the-aria-controls-attribute"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-is-the-aria-describedby-attribute",
+      "destination": "/learn/full-stack-developer/lecture-introduction-to-aria/what-is-the-aria-describedby-attribute"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/when-is-the-alt-attribute-needed",
+      "destination": "/learn/full-stack-developer/lecture-accessible-media-elements/when-is-the-alt-attribute-needed"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-are-the-accessibility-benefits-for-good-link-text",
+      "destination": "/learn/full-stack-developer/lecture-accessible-media-elements/what-are-the-accessibility-benefits-for-good-link-text"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-are-good-ways-to-make-audio-and-video-content-accessible",
+      "destination": "/learn/full-stack-developer/lecture-accessible-media-elements/what-are-good-ways-to-make-audio-and-video-content-accessible"
+    },
+    {
+      "source": "/learn/full-stack-developer/lecture-importance-of-accessibility-and-good-html-structure/what-are-some-ways-to-make-web-applications-keyboard-accessible",
+      "destination": "/learn/full-stack-developer/lecture-accessible-media-elements/what-are-some-ways-to-make-web-applications-keyboard-accessible"
```

Comment:
something went wrong here, this object has duplicated keys

Reply:
Fixed.